.github/workflows/ci.yaml

```diff
@@ -273,6 +321,18 @@ jobs:
                   ASPECT_NPM_AUTH_TOKEN: ${{ secrets.ASPECT_NPM_AUTH_TOKEN }}
                   ASPECT_GH_PACKAGES_AUTH_TOKEN: ${{ secrets.ASPECT_GH_PACKAGES_AUTH_TOKEN }}
 
+            - name: bazel test //... (Windows)
```

Comment:
This just drops the `ls | ... | grep`? Was that causing issues worth copy+pasting this for windows?